### DISCUSSION TITLE
Установка и вход в админку

### DISCUSSION THREAD

**Discussion by xstulovox** [maxsite/cms#439] [Q&A]
Здравствуйте всем!
После чистой установки MS открылась главная страница нового сайта - http://prntscr.com/10vsl90
Далее, при попытке перейти на страницу входа в админку (через ссылку в подвале), выходит ошибка: http://prntscr.com/10vso0l
Not Found
The requested URL was not found on this server.
Additionally, a 404 Not Found error was encountered while trying to use an ErrorDocument to handle the request.

Такая же ошибка выходит при попытке перейти на любую страницу из меню. 

Как же всё-таки зайти в админку? И почему такая ошибка появляется?

P.S. Права на uploads и cashe стоят 0777.

**Comment by maxsite** [comment]
Не работает ЧПУ. Смотрите файл .htaccess. Если не работает mod_rewrite, то обращайтесь к хостеру.

**Reply by xstulovox** [reply]
Спасибо! Да, всё дело в .htaccess было.
